Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -88,7 +201,34 @@ public static int[] InsertLast(int[] array, int number)
         /// <returns>A new array with element inserted at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertAt(int[] array, int number, int index)
         {
-            // ToDo: implement.
+            if (array != null && array.Length != 0)
+            {
+                var arrayLength = array.Length;
+                if (index < 0 || index > arrayLength) return array;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011611)